packages/plugins/src/sample-project/sample-contracts.ts

```diff
@@ -36,10 +36,11 @@ export const getSampleScriptFile = (
   return `// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import { Script } from "forge-std/Script.sol";
 import { HelloSphinx } from "${relativeSrcPath}/HelloSphinx.sol";
 import "@sphinx-labs/contracts/SphinxPlugin.sol";
 
-contract HelloSphinxScript is Sphinx {
+contract HelloSphinxScript is Sphinx, Script {
```

Comment:
The `Solc.spec.ts` test suite was failing when I added the `--sizes` flag to the `forge build` commands. This was happening because `HelloSphinxScript` is over the size limit and because it doesn't inherit from Foundry's `Script.sol`, which contains an `IS_SCRIPT` function. The `forge build --sizes` command skips contracts that have the `IS_SCRIPT` function ([source](https://github.com/foundry-rs/foundry/issues/4615)). We now inherit from `Script.sol` to resolve this issue, and because it's standard for Forge script to inherit from this contract